syntax="proto3";

message PointMes
{
    double lat = 1;
    double lon = 2;
}